pkg/services/ngalert/schedule/testing.go

```diff
@@ -81,3 +85,26 @@ func (f *fakeRulesStore) DeleteRule(rules ...*models.AlertRule) {
 func (f *fakeRulesStore) getNamespaceTitle(uid string) string {
 	return "TEST-FOLDER-" + uid
 }
+
+type SyncAlertsSenderMock struct {
```

Comment:
Some tests were racing while accessing `(*AlertsSenderMock).Mock.Calls`, so I added this mutex-protected version.